WSRequestHandler.cpp

```diff
@@ -1093,4 +1095,286 @@ void WSRequestHandler::HandleGetRecordingFolder(WSRequestHandler* req)
 
 	req->SendOKResponse(response);
 	obs_data_release(response);
-}
\ No newline at end of file
+}
+
+void WSRequestHandler::HandleGetTextGDIPlusProperties(WSRequestHandler* req)
+{
+
+	const char* itemName = obs_data_get_string(req->data, "source");
+	if (!itemName)
+	{
+		req->SendErrorResponse("invalid request parameters");
+		return;
+	}
+
+	const char* sceneName = obs_data_get_string(req->data, "scene-name");
+	obs_source_t* scene = Utils::GetSceneFromNameOrCurrent(sceneName);
+	if (scene == NULL) {
```

Comment:
obs-websocket uses C+11, so please considering the use of `nullptr` instead of `NULL`. You can also write `!scene` instead of `scene == NULL`.